vue-frontend/src/pages/contact.vue

```diff
@@ -74,6 +90,10 @@ export default {
     setLoader(loading) {
       this.isLoading = loading;
     },
+    GetData(data) {
```

Comment:
Method name should start with small letters and give the appropriate name like `showBadge` or `showSuccessMessage`. 
I think the `type` field is not required